src/frontend/src/lib/components/hero/about/AboutHow.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+	import AboutItem from '$lib/components/hero/about/AboutItem.svelte';
+	import { i18n } from '$lib/stores/i18n.store';
+	import { modalStore } from '$lib/stores/modal.store';
+	import { replaceOisyPlaceholders } from '$lib/utils/i18n.utils';
+	import { createEventDispatcher } from 'svelte';
+
+	export let asMenuItem = false;
+
+	const dispatch = createEventDispatcher();
+
+	const openModal = () => {
+		modalStore.openAboutHow();
+		dispatch('click');
```

Comment:
Should we maybe inverse the order - i.e. dispatch before open modal - to have two backdrop at the top of each other for a shorter period?

If we inverse the order, should we keep 'click' or give it a particular name like `dispatch('icOpenModal')`?

Reply:
- inverse the order: yes, done
- change name: good question... to be consistent, I would say yes: `icOpenAboutModal` ?

Reply:
`icOpenAboutModal` sounds cool if that works and makes sense for you?